src/main/java/blackjack/domain/gambler/Gambler.java

```diff
@@ -0,0 +1,74 @@
+package blackjack.domain.gambler;
+
+import blackjack.domain.WinningDiscriminator;
+import blackjack.domain.card.Card;
+import blackjack.domain.card.CardType;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public abstract class Gambler {
+    private static final int MIN_ACE_VALUE = 1;
+    private final List<Card> cards;
+    private final Name name;
+
+    public Gambler(final Name name) {
+        this.name = name;
+        this.cards = new ArrayList<>();
+    }
+
+    public void addCard(final Card card) {
+        cards.add(card);
+    }
+
+    public boolean isNameEquals(final Name name) {
+        return Objects.equals(name, this.name);
+    }
+
+    public boolean isScoreBelow(final int criteria) {
+        return calculateScore() <= criteria;
+    }
+
+    public int calculateScore() {
+        int sum = calculateSum();
+        long aceCount = calculateAceCount();
+        if (aceCount == 0) {
+            return sum;
+        }
+        return adjustSumByAce(sum, (int) aceCount);
+    }
+
+    private int calculateSum() {
+        return cards.stream()
+                .mapToInt(Card::getTypeValue)
+                .sum();
+    }
+
+    private long calculateAceCount() {
+        return cards.stream()
+                .filter(Card::isAce)
+                .count();
+    }
+
+    protected int adjustSumByAce(int sum, int aceCount) {
+        int differenceBetweenMaxAndMinValue = CardType.ACE.getValue() - MIN_ACE_VALUE;
+        sum -= differenceBetweenMaxAndMinValue * (aceCount - 1);
+
+        if (sum > WinningDiscriminator.BLACK_JACK) {
+            sum -= differenceBetweenMaxAndMinValue;
+        }
+        return sum;
+    }
```

Comment:
```suggestion
    protected int adjustSumByAce(int sum, int aceCount) {
        int minSum = minSum(sum, aceCount);
        if (minSum > WinningDiscriminator.BLACK_JACK) {
            return minSum - differenceBetweenMaxAndMinValue();
        }
        return minSum;
    }

    private int minSum(int sum, int aceCount) {
        return sum - differenceBetweenMaxAndMinValue() * (aceCount - 1);
    }

    private int differenceBetweenMaxAndMinValue() {
        return CardType.ACE.getValue() - MIN_ACE_VALUE;
    }
```

중간에 if문에서 흐름이 끝나지 않다보니 놓치기 쉬울 것 같네요. 개인적으로 가능하면 if문에서 흐름을 끝낼 수 있도록 하는걸 선호합니다. 위와 같이 메서드를 추출해보면 되겠네요~